webserver/src/test/java/io/kestra/webserver/controllers/ExecutionControllerTest.java

```diff
@@ -52,6 +53,8 @@
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
 class ExecutionControllerTest extends JdbcH2ControllerTest {
+    public static final String URL_LABEL_VALUE = "https://some-url.com";
+    public static final String ENCODED_URL_LABEL_VALUE = URL_LABEL_VALUE.replace("/", URLEncoder.encode("/", StandardCharsets.UTF_8));
```

Comment:
```suggestion
    public static final String ENCODED_URL_LABEL_VALUE = URLEncoder.encode(URL_LABEL_VALUE, StandardCharsets.UTF_8));
```

The all string must be encoded

Reply:
That's what I did before, I forgot a comment but I saw that this is the behaviour of our UI library (for eg. ":" aren't encoded from UI) so I prefer to replicate this encoding to ensure we don't miss any side-effect (":" parsing in this example)

Reply:
We should URL encode everything from the UI, this is a bug and a security risk I think